src/devana/syntax_abstraction/variable.py

```diff
@@ -124,6 +137,19 @@ def from_cursor(cls, cursor: cindex.Cursor, parent: Optional = None) -> Optional
         result = cls(cursor, parent)
         return result
 
+    @classmethod
+    @init_params(skip={"cls", "parent"})
```

Comment:
If we almost always ignore cls and parent maybe we can get rid of that parameter and add it to the implementation?

Reply:
I thought of that too, but then it would be less flexible as a whole. Alternatively, this could be added as a default ignored parameters.
Something like this `ignore = skip or set("skip", "parent")`

Reply:
Specifying `cls` only should be ok, after all this is mainly intended for classmethods. However, putting also `parent` is meh, in tests this is unnecessary and in two cases in "normal code" also.